twilio_test.go

```diff
@@ -7,7 +7,7 @@ import (
 )
 
 func TestClient_WithParams(t *testing.T) {
-	client := NewRestClientWithParams(RestClientParams{
+	client := NewRestClientWithParams(ClientParams{
```

Comment:
Would it be worthwhile to add tests for the use of subaccounts?

Reply:
I thought this test was testing the use of subAccountSid vs the other one is testing the main account?

Reply:
Ah alright, disregard then